app/v2/models/request/common/ukNonFhlProperty/UkNonFhlPropertyIncome.scala

```diff
@@ -38,6 +37,15 @@ object UkNonFhlPropertyIncome {
       (JsPath \ "taxDeducted").writeNullable[BigDecimal] and
       (JsPath \ "otherIncome").writeNullable[BigDecimal] and
       (JsPath \ "ukOtherRentARoom").writeNullable[UkPropertyIncomeRentARoom]
-  ) (unlift(UkNonFhlPropertyIncome.unapply))
+  )(unlift(UkNonFhlPropertyIncome.unapply))
+
+  val writesIFS: Writes[UkNonFhlPropertyIncome] = (
```

Comment:
Is this meant to have both writes and writesIFS?

Reply:
Can the // DONE comments be removed?

Reply:
> Is this meant to have both writes and writesIFS?

They do differ slightly - lines 48 and 62 are where the two writes differ. Open to a less verbose way of achieving this?

Reply:
In this case I think more verbose is going to be the less risky answer :disappointed:
(see Slack comments)